app/src/main/java/de/tum/in/tumcampusapp/component/ui/overview/SupportCard.java

```diff
@@ -26,31 +26,34 @@ public SupportCard(Context context) {
     public static CardViewHolder inflateViewHolder(ViewGroup parent) {
         final View view = LayoutInflater.from(parent.getContext())
                                         .inflate(R.layout.card_support, parent, false);
-        //Add links to imageviews
-        view.findViewById(R.id.facebook)
+
+        view.findViewById(R.id.facebook_button)
             .setOnClickListener(v -> {
                 Intent browserIntent = new Intent(Intent.ACTION_VIEW);
                 browserIntent.setData(Uri.parse(view.getContext()
                                                     .getString(R.string.facebook_link)));
-                view.getContext()
-                    .startActivity(browserIntent);
+                v.getContext()
+                 .startActivity(browserIntent);
             });
-        view.findViewById(R.id.github)
+
+        view.findViewById(R.id.github_button)
             .setOnClickListener(v -> {
                 Intent browserIntent = new Intent(Intent.ACTION_VIEW);
                 browserIntent.setData(Uri.parse(view.getContext()
                                                     .getString(R.string.github_link)));
-                view.getContext()
-                    .startActivity(browserIntent);
+                v.getContext()
+                 .startActivity(browserIntent);
             });
-        view.findViewById(R.id.email)
+
+        view.findViewById(R.id.email_button)
             .setOnClickListener(v -> {
-                Intent mailIntent = new Intent(Intent.ACTION_SENDTO);
```

Comment:
Why change this Intent? Before it sent an email to 'app@tum.de' with an email app of the users choosing. Now there is no email address specified and you can choose from a lot of unfitting apps for sending an email. We should probably use the code from before?

Reply:
That was not my… intent ;) I think I accidentally changed something here. Restored it to the old behavior.